doc/metrics.md

```diff
@@ -0,0 +1,559 @@
+
+```typescript
+const dropView = new View({
+  aggregation: new DropAggregation(),
+  instrumentName: 'htpp*',
+});
+```
+
+### Customizing the metric attributes of instrument
+
+If you want to limit the Metric Attributes that get exported in measurements of
+an instrument, you can create a Metric View which defines which attributes should
+be exported. Attributes that are missing in the list will not be exported.
+
+In the example below will drop all attributes except attribute `environment` for
+all instruments.
+
+```typescript
+new View({
+  // only export the attribute 'environment'
+  attributeKeys: ['environment'],
+  // apply the view to all instruments
+  instrumentName: '*',
+})
+```
+
+## Exporting measurements
+
+After you have instrumented your application with metrics, you also need to make
+sure that the metrics get collected by your metrics backend. The most common formats
+that are used are Prometheus and OLTP.
+
+The latter is the [Opentelemetry protocol format](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md)
+which is supported by the Opentelemetry Collector. The former is based on the [OpenMetrics
+format](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md) can be consumed by Prometheus and Thanos or other OpenMetrics compatible
+backends.
+
+_Note_: Both Opentelemetry Javascript and Opentelemetry Collector support
+exporters for different formats, such as [Cloud Monitoring](https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/tree/master/packages/opentelemetry-cloud-monitoring-exporter).
+
+## Exporting measurements to Prometheus
+
+If you want to export your metrics to Prometheus you need to initialize Opentelemetry
+to use the Prometheus exporter `PrometheusExporter` which is included in the
+`@opentelemetry/exporter-prometheus`-package.
+
+```typescript
+const { PrometheusExporter } = require('@opentelemetry/exporter-prometheus');
+const { MeterProvider }  = require('@opentelemetry/sdk-metrics');
+
+// Add your port and startServer to the Prometheus options
+const options = { port: 9464, startServer: true };
+const exporter = new PrometheusExporter(options);
+
+// Creates MeterProvider and installs the exporter as a MetricReader
+const meterProvider = new MeterProvider();
+meterProvider.addMetricReader(exporter);
+const meter = meterProvider.getMeter('example-prometheus');
+
+// Now, start recording data
+const counter = meter.createCounter('metric_name', {
+  description: 'Example of a counter'
+});
+counter.add(10, { pid: process.pid });
+```
+
+In the above example the instantiated `PrometheusExporter` is configured to expose
+a new http server on port 9464. You can now access the metrics at the endpoint
+<http://localhost:9464/metrics>.  This is the url that can be scraped by Prometheus so it can consumed the metrics collected by Opentelemetry in your application.
```

Comment:
```suggestion
<http://localhost:9464/metrics>. This is the URL that can be scraped by Prometheus so it can consumed the metrics collected by OpenTelemetry in your application.
```